### ISSUE TITLE
Trakt account hasn't been authorised

### ISSUE THREAD

**Issue by DropbearNinja** [trakt/Plex-Trakt-Scrobbler#402]
Sorry, through searching I know others have had this issue and solved, but I can't for the life of me figure it out.

I cannot sync data at all. 

PMS Version 1.3.3.3148, Plex web (signed in) Version 2.12.5, i've included the pin in trakt settings. I've tried re-installing. Both manually and through unofficial installer (webtools). 

[com.plexapp.plugins.trakttv.log.zip](https://github.com/trakt/Plex-Trakt-Scrobbler/files/737410/com.plexapp.plugins.trakttv.log.zip)

**Comment by DropbearNinja** [comment]
So I've discovered that it doesn't connect because of my VPN - turning off my VPN allows it to work.

**Comment by fuzeman** [comment]
Your log file doesn't show the actual authentication error, it only shows that your Trakt.tv account hasn't currently been authenticated. Could you upload the latest plugin log file right after attempting to authenticate again?

------------

> Any advice on getting it to work with vpn?

The plugin should work with TAP/TUN VPNs (e.g. OpenVPN) without any configuration changes.

If your VPN/Proxy service uses HTTP(s) or SOCKS proxies, you will need to configure the plugin to use it via the settings in Plex/Web.

**Comment by DropbearNinja** [comment]
To create that log file I cleared the file. Rebooted. Waited to ensure everything was loaded, then ran a full sync. Don't know why it wouldn't have a authentication attempt?

This log file contains probably too much, but it also contains (I think. In log.3) the period in which I disabled my vpn and it worked, then re-enabled vpn and it stopped working. 
[trakt log.zip](https://github.com/trakt/Plex-Trakt-Scrobbler/files/741939/trakt.log.zip)

**Comment by fuzeman** [comment]
> To create that log file I cleared the file. Rebooted. Waited to ensure everything was loaded, then ran a full sync. Don't know why it wouldn't have a authentication attempt?

The plugin only authenticates when you enter a new pin or login via the [configuration site](http://trakt-for-plex.github.io/configuration).

> This log file contains probably too much, but it also contains (I think. In log.3) the period in which I disabled my vpn and it worked, then re-enabled vpn and it stopped working.

Based on your log files it appears that the plugin ran one sync successfully after you authenticated your account, but then started failing again for no obvious reason.. looks like an issue with the handling of authentication tokens in the plugin.

------------

Could you try authenticate your account again by entering a new pin into the plugin settings (via Plex/Web) and post the latest log file here again.

**Comment by DropbearNinja** [comment]
So I managed to fix the problem.

For anyone who has this problem and looking for a solution: I'm running through a VPN, but had my DNS servers set to google's public DNS. This, I've discovered, actually caused a couple of issues I had, including getting intermittent catpcha requests from cloudfare. Setting my DNS servers to my VPNs servers (or allowing auto) fixed the problem. I can now scrobble through my VPN and also fixed a few other things.

My assumption is that having data come from a different IP to the DNS request is a trigger for servers to think you're up to no good...

Thanks for your assistance.
